holidays/countries/hongkong.py

```diff
@@ -151,67 +155,52 @@ def _populate_public_holidays(self):
         if self._year <= 1962:
             return None
 
-        # The first day of January.
-        # 一月一日.
         if self._year >= 1977:
-            self._add_observed(self._add_new_years_day("The first day of January"))
+            # The first day of January.
+            self._add_observed(self._add_new_years_day(tr("一月一日")))
 
         self._add_lunar_new_year(day_three_start_year=1977)
 
-        # Good Friday.
-        # 耶穌受難節.
         if self._year >= 2028:
-            self._add_good_friday("Good Friday")
+            # Good Friday.
+            self._add_good_friday(tr("耶穌受難節"))
 
-        # The day following Good Friday.
-        # 耶穌受難節翌日.
         if self._year >= 2030:
-            self._add_holy_saturday("The day following Good Friday")
+            # The day following Good Friday.
+            self._add_holy_saturday(tr("耶穌受難節翌日"))
 
-        # Easter Monday.
-        # 復活節星期一.
         if self._year >= 2026:
-            self._add_easter_monday("Easter Monday")
+            # Easter Monday.
+            self._add_easter_monday(tr("復活節星期一"))
 
         # Ching Ming Festival.
-        # 清明節.
-        self._add_observed(self._add_qingming_festival("Ching Ming Festival"))
+        self._add_observed(self._add_qingming_festival(tr("清明節")))
 
-        # The Birthday of the Buddha.
-        # 佛誕.
         if self._year >= 2022:
-            self._add_observed(self._add_chinese_birthday_of_buddha("The Birthday of the Buddha"))
+            # The Birthday of the Buddha.
+            self._add_observed(self._add_chinese_birthday_of_buddha(tr("佛誕")))
 
-        # Labour Day.
-        # 勞動節.
         if self._year >= 1999:
-            self._add_observed(self._add_labor_day("Labour Day"))
+            # Labour Day.
```

Comment:
```suggestion
            # Labor Day.
```
For standardization, as we traditionally write msg comments in `en_US` locale (here and in 2 locations below).